portal-api/src/v1/gef/services/clone/helpers/generate-cloned-deal.ts

```diff
@@ -0,0 +1,85 @@
+import { ObjectId } from 'mongodb';
+import { produce } from 'immer';
+import { AuditDetails, Bank, getCurrentGefDealVersion, PortalSessionUser, GefDeal, AnyObject } from '@ukef/dtfs2-common';
+import { generateAuditDatabaseRecordFromAuditDetails } from '@ukef/dtfs2-common/change-stream';
+import { DEAL } from '../../../../../constants';
+import { generateClonedExporter } from './generate-cloned-exporter';
+
+const propertiesToRemove: (keyof GefDeal)[] = [
```

Comment:
Can we export this and add a unit test please?

Reply:
This is a constant - what sort of unit test are you expecting? These are tested by `it('removes the expected properties', () => {`

Reply:
Concern here is that a property could be added here and the tests would still pass. If we test this specifically, it'll fail if a property is added.

Reply:
Bump